external-crates/move/crates/move-analyzer/trace-debug/README.md

```diff
@@ -1,20 +1,42 @@
 
 # How to Install
 
-When this extension and its companion DAP implementation become more mature, we will make it available in the VSCode Marketplace. At the moment, in order to experiment with it must be built and installed locally (start in the main directory of this extension):
-1. Install dependencies for the extension:
-```bash
-npm install
-```
-2. Install dependencies for the DAP implementation:
-```bash
-npm install --prefix ../move-trace-adapter
-```
-3. Package the extension (it will pull relevant files from the DAP implementation)
-```
-vsce package -o move-trace-debug.vsix
+1. Open a new window in any Visual Studio Code application version 1.61.0 or greater.
+2. Open the command palette (`⇧` + `⌘` + `P` on macOS, or use the menu item *View > Command Palette...*) and
+   type **Extensions: Install Extensions**. This will open a panel named *Extensions* in the
+   sidebar of your Visual Studio Code window.
+3. In the search bar labeled *Search Extensions in Marketplace*, type **Move Trace**. The Move Trace debugger extension
+   should appear as one of the option in the list below the search bar. Click **Install**.
+
+# How to trace-debug a Move unit test
+
+Debugging a Move unit tests consists of two steps: generating a Move trace and actually trace-debugging it
+
+## Generating a Move trace
+
+If you have [Mysten's Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) installed you can generate a Move trace for tests defined in a given file by navigating to this file in VSCode and running `Move: Trace Move test execution` from the command palette. See the description of [Mysten's Move extension](https://marketplace.visualstudio.com/items?itemName=mysten.move) for pre-requisites needed to run this command.
+
+If you plan to use the the Trace Debugging Extension by itself, you need to generate the traces using command-line interface of `sui` binary. See [here](https://docs.sui.io/guides/developer/getting-started/sui-install) for instructions on how to install `sui` binary. Note that the `sui` binary must be built with the `tracing` feature flag. If your version of the `sui` binary was not built with this feature flag, an attempt to trace test execution will fail. In this case you may have to build the `sui` binary from source following these [instructions](https://docs.sui.io/guides/developer/getting-started/sui-install#install-sui-binaries-from-source).
+
+Once the `sui` binary is installed, you generate traces for all test files in a given package by running the following command in the package's root directory:
+```shell
+sui move test OPTIONAL_FILTER --trace-execution
```

Comment:
Could we, rather than the OPTIONAL_FILTER here, have another command stanza that explains "If you want to filter the tests that are traced, use this filter to run and trace only the tests matching FILTER."

This just cleans it up and makes it less ambiguous how everything works.

Reply:
Happy to change but want to better understand the intended changed. Are you suggesting to have two CLI examples, one with and one without the filter?

Reply:
Went with what guessed. Can certainly finessed it further at a later time